### PR TITLE
ORC-1100: Add vcpkg release document

### PR DESCRIPTION
**Pull request by deshanxiao**
### What changes were proposed in this pull request?
This PR is aimed to add vcpkg document to note how to update vcpkg package when we release a new version.

### Why are the changes needed?
To help release manager to release our latest package in vcpkg.

### How was this patch tested?
N/A

### Was this patch authored or co-authored using generative AI tooling?
No

### CODE REVIEW

**Comment by deshanxiao** [comment]
cc @dongjoon-hyun @wgtmac

File: site/develop/make-release.md [hunk 1]
```diff
@@ -188,3 +188,51 @@ Update ORC's jira to reflect the released version.
...
+Update the **version** field of [vcpkg.json](https://github.com/microsoft/vcpkg/blob/master/ports/orc/vcpkg.json). Run the following command to rebuild ORC:
+```
+% ./vcpkg build  orc
+```
+
+You will receive an error, which contains a SHA512 value, updating it to the [portfile.cmake](https://github.com/microsoft/vcpkg/blob/master/ports/orc/portfile.cmake) file.
+
+> Expected hash: 141afbd6d83b8c8032df071838e7da61eedb3d22289642f76669b6efd167d7550b200bd3542f012d0b63c9ae2572d83fcb1b7f76537b6fa6f980aebf37e2cde2
+
+> Actual hash (you should update): 7b9d6f9613f48b41843618465c0c71ed70e5695b4fc4b3db15d0bbfe4c5126e0c6c8f1114d6c450a56b4ddf0357d89bd40cb03c0964275c59761cc0f8fec1291
```
Comment (mystic-lama) [inline site/develop/make-release.md:215]: Is the actual hash constant or we get it from somewhere?

Don't know much about vcpkg

**Comment by deshanxiao** [comment]
@dongjoon-hyun yes, I got the hash from the error message. From the vcpkg document, it recommand us to update it to zero and get the hash from error message too.

**Comment by deshanxiao** [comment]
For orc 1.9.2:

141afbd6d83b8c8032df071838e7da61eedb3d22289642f76669b6efd167d7550b200bd3542f012d0b63c9ae2572d83fcb1b7f76537b6fa6f980aebf37e2cde2

**Comment by dongjoon-hyun** [comment]
Thank you, @deshanxiao and all.

Merged to main/2.0. We can revise it later if needed.

**Pull request closed without merging**